### PR TITLE
feat: update entity lookup method for pagination from endpoint

### PR DESCRIPTION
**Pull request by isaackps**
In this PR: 
- Update entity lookup method with pagination to api.
- update the return response with the identities, count and total from the API.

### CODE REVIEW

File: src/services/addressResolver/index.ts [hunk 1]
```diff
@@ -49,14 +46,19 @@ const get = async ({
...
-}: EntityLookupProps): Promise<AddressBookThirdPartyResultsProps[]> => {
-  const url = `${endpoint}search?q=${query}`;
+}: EntityLookupProps): Promise<EntityLookupResponseProps> => {
+  const anyLimit = limit !== undefined ? `&limit=${limit}` : "";
+  const anyOffset = offset !== undefined ? `&offset=${offset}` : "";
+
+  const url = `${endpoint}search?q=${query}${anyOffset}${anyLimit}`;
   const response = await get({ url, apiHeader, apiKey });
-  return response.data.identities;
+  return response.data;
```
Comment (simboonlong) [inline src/services/addressResolver/index.ts]: want to use query-string library? then u can do:

```
  const url = queryString.stringifyUrl({
    url: `${endpoint}search`,
    query: {
      q: query,
      limit,
      offset,
    },
  });
```

File: package.json [hunk 2]
```diff
@@ -26,6 +27,7 @@
     "test": "jest --ci",
     "test:coverage": "npm run test -- --coverage",
     "test:watch": "npm run test -- --watch",
+    "prep": "npm-run-all -s lint test build",
```
Comment (simboonlong) [inline package.json]:
```suggestion
    "precommit": "run-s lint test build",
```

Comment (isaackps) [reply package.json]: ok

File: src/services/addressResolver/index.ts (hunk 1 above)
Comment (isaackps) [reply src/services/addressResolver/index.ts]: ok

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**